apps/condo/domains/ticket/utils/shareDataPacker.ts

```diff
@@ -73,16 +75,18 @@ export function unpackShareData (data: string): string {
     }
 }
 
-export function packShareData (data: string): string {
+export function packShareData (data: string, encode: Encode = INFLATE_MARKER): string {
     // New version using AES-CTR and Deflate compression
     // Using KEY directly as IV
     const iv = Buffer.from(KEY).subarray(0, IV_LENGTH)
     const cipher = crypto.createCipheriv(MODERN_ALGORITHM, Buffer.from(KEY), iv)
-
+    console.log('zlib', zlib)
     // NOTE: it's more efficient but its require modern node version! please use it in a future when we will use useModern = true by default!
```

Comment:
It's more efficient but requires modern node version (which one?)

Reply:
Actually, 22 is enough, but it breaks in browser environments, since its polyfills does not provide brotli

Reply:
Yea, this is needed to be stated in the comment